vsts/node-release.yaml

```diff
@@ -0,0 +1,45 @@
+name: Release_$(BuildID)_$(Date:yyyyMMdd)$(Rev:.r)
+resources:
+- repo: self
+  clean: true
+
+phases:
+
+- phase: Phase_1
+  displayName: Create Release Branch
+
+  condition: succeededOrFailed()
+  queue:
+    name: Hosted Ubuntu 1604
+
+  steps:
+
+  - checkout: self  # self represents the repo where the initial Pipelines YAML file was found
+    persistCredentials: true  # set to 'true' to leave the OAuth token in the Git config after the initial fetch
+    clean: true
+
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '10.x'
+
+  - task: Npm@1
+    inputs:
+      command: 'install'
+      workingDir: $(Build.SourcesDirectory)/scripts/version
+
+  - powershell: $(Build.SourcesDirectory)/scripts/bump-versions.ps1
+    displayName: Bump Versions
+
+  - task: PowerShell@2
```

Comment:
>- task: PowerShell@2 [](start = 2, length = 20)

you're missing a mandatory step before this.
after bumping versions you still need to run `lerna bootstrap` and `lerna run build` to generate the javascript.

my comment about removing other (disabled) steps were about `lerna run ci` and `lerna run e2e`. these are the ones we don't need. we still need `lerna bootstrap` and `lerna run build`